Yoast/wordpress-seo
# Hide indexation warning for non admins

## Context


* The indexation warning is showing for everybody, it shouldn't be.

## Summary


This PR can be summarized in the following changelog entry:

* Hides the indexables indexation warning for non-admin roles.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Make sure you have a site that needs indexation.
* Switch to a non-admin role, see you get the indexation warning.
* Check out this pull.
* See the warning is gone for non-admins.

## UI changes

## Documentation

## Quality assurance


Fixes https://github.com/Yoast/featurerequests/issues/544
